src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -3827,7 +3827,7 @@ void resetCurrentCommand() throws SQLServerException {
         }
     }
 
-    boolean isConnectionDead() throws SQLServerException {
+    public boolean isConnectionDead() throws SQLServerException {
```

Comment:
We definitely don't want this public. You can access it via reflection in the test.